README.md

```diff
@@ -572,116 +538,15 @@ Now, any script you defined in the ScriptsFunction will be added to the HTML tog
 
 > Tip: You could use it together with useShouldHydrate to disable Remix scripts in certain routes but still load scripts for analytics or small features that need JS but don't need the full app JS to be enabled.
 
-### StructuredData
+### useGlobalNavigationState
 
-> **Warning**: Deprecated in favor of the V2_MetaFunction. This will be removed in the next major version. Check below for the new way to do this.
-
-If you need to include structured data (JSON-LD) scripts on certain routes, you can use the `StructuredData` component together with the `HandleStructuredData` type or `StructuredDataFunction` type.
-
-In the route you want to include the structured data, add a `handle` export with a `structuredData` method, this method should implement the `StructuredDataFunction` type.
+This hook lets you know if the value of `transition.state`, every `fetcher.state` in the app, and `revalidator.state`.
```

Comment:
```suggestion
This hook allows you to read the value of `transition.state`, every `fetcher.state` in the app, and `revalidator.state`.
```

Reply:
not sure about this one, but the original sentence definitely needs to be rewritten